utils/hare-bootstrap

```diff
@@ -275,12 +291,12 @@ if sudo systemctl --quiet is-active hare-consul-agent; then
     say 'Stopping Consul on other nodes...'
     pids=()
     while read node bind_ip; do
-        ssh $node "sudo systemctl stop hare-consul-agent" &
+        ssh_safe "sudo systemctl stop hare-consul-agent" &
```

Comment:
same here, don't you need to pass `$node` here as well?

Reply:
I don't think so it explicitly needs to be passed. It can access the $node in that while do logic. the jq - ... if i'm not wrong. Still i'll test it myself.

Reply:
Altered the ssh function to take explicit args and refined the script.